### ISSUE TITLE
VPAT 2.4.4 Link Purpose-General Site Updates, Not Permit Finder Specific

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#20]
Screengrab/videos (insert file names here)
1. Places link.png
2. Government.png
3. City of Boston Seal link.png

Link
https://permit-finder.digital-staging.boston.gov

Environment
(Device > OS > Browser)
Computer > macOS 10.14.6 > Safari 12.1.2

Brief Description
Some links need additional description

Steps to reproduce
1. The Places and Government links on the submenu are both linked to https://www.boston.gov/#. These links are not consistent with the items listed in these categories.
2. The Boston Seal links to https://www.boston.gov, but this is not clear from the accessibility label.

Desired Result
Links are clearly identified for accessibility users.

Actual Result
Additional description is needed.
![3  City of Boston Seal link](https://user-images.githubusercontent.com/33845414/62467060-e3512480-b760-11e9-96bf-5057a7132841.png)
![1  Places link](https://user-images.githubusercontent.com/33845414/62467061-e3512480-b760-11e9-9621-990a0334afea.png)
![2  Government](https://user-images.githubusercontent.com/33845414/62467062-e3512480-b760-11e9-88ef-da2548607d48.png)

**Comment by reillyzlab** [comment]
Probably don't need if moving into boston.gov

**Comment by paylakatel** [comment]
This squarely a patterns/D8 issue - not permit finder, we're just pulling in this source code. I think it should be addressed when the entire patterns library is looked at.

**Comment by subaha-cob** [comment]
Fixed on Drupal site. Added Title tags.
https://d8-uat.boston.gov/
Permit Finder is a separate App. I will not be going into the App to fix it. 
All fixes are restricted to Boston.gov.
Patterns fixes will be necessary and that should help some of the APPS using the library.

**Comment by subaha-cob** [comment]
@andrebtlima I can write in content for the items above. But you can also provide the content for me to use.

**Comment by subaha-cob** [comment]
Test on: https://d8-uat.boston.gov/

**Comment by jill-iterators** [comment]
Verified in UAT, we will do regression test on links later in order to finalize the site.
